xapian-core/tests/harness/backendmanager_honey.cc

```diff
@@ -73,12 +73,50 @@ BackendManagerHoney::do_get_database_path(const vector<string> & files)
     return dbpath;
 }
 
+Xapian::WritableDatabase
+BackendManagerHoney::get_generated_database(const string& name)
+{
+    // creating generated database inside glass cache
+    // to prevent a valid glass db inside honey cache
+    // if testsuite was interrupted
+    return generated_sub_manager->get_generated_database(name);
```

Comment:
"Create ..." would read better and make this a complete sentence.  Similarly "Covert ..." just below.

Reply:
Changed appropriately.